packages/data/src/options/resolvers.js

```diff
@@ -1,25 +1,38 @@
 /**
  * External Dependencies
  */
-
 import { apiFetch } from '@wordpress/data-controls';
+import { batchFetch } from './controls';
 
 /**
  * Internal dependencies
  */
 import { WC_ADMIN_NAMESPACE } from '../constants';
-import { receiveOptions, setIsRequesting, setRequestingError } from './actions';
+import { receiveOptions, setRequestingError } from './actions';
 
 export function* getOptionsWithRequest( names ) {
-	yield setIsRequesting( true );
 	const url = WC_ADMIN_NAMESPACE + '/options?options=' + names.join( ',' );
 
 	try {
 		const results = yield apiFetch( { path: url } );
 		yield receiveOptions( results );
 		return results;
 	} catch ( error ) {
-		yield setRequestingError( error );
+		yield setRequestingError( error, names );
 		return error;
 	}
 }
+
+/**
+ * Request an option value.
+ * 
+ * @param {string} name - Option name
+ */
+export function* getOption( name ) {
+	try {
+		const result = yield batchFetch( name );
+		yield receiveOptions( result );
+	} catch ( error ) {
+		yield setRequestingError( error, name );
```

Comment:
I'm not sure this will be entirely accurate. With the batch fetch, there's the potential for multiple `names` to be in the request that causes the error right? Would you want that reflected in the error state? That's why in my example I suggested yielding an object from the batchFetch control: `{ result, names }` so you have all the option names that were used in the _actual_ request.